sdk/scheduler/src/main/java/com/mesosphere/sdk/curator/CuratorPersister.java

```diff
@@ -313,61 +313,76 @@ public void setMany(Map<String, byte[]> unprefixedPathBytesMap) throws Persister
     }
 
     @Override
+    public Map<String, byte[]> getMany(Collection<String> unprefixedPaths) throws PersisterException {
+        if (unprefixedPaths.isEmpty()) {
+            return Collections.emptyMap();
+        }
+        logger.debug("Getting many entries: {}", unprefixedPaths);
```

Comment:
For debugging it'd probably be useful to actually say how many here? Similar to above log like this.

Reply:
Good point, added to both.